src/knex-tables.d.ts

```diff
@@ -320,7 +320,6 @@ declare module "knex/types/tables" {
   interface OnerepProfileRow {
     id: number;
     onerep_profile_id: null | number;
-    name_suffix: null | string;
```

Comment:
I think I missed the memo on this, are we deprecating the `name_suffix` field entirely?

Reply:
No, we are keeping it for now: [41912ff](https://github.com/mozilla/blurts-server/pull/5660/commits/41912ffc106f670f6500e6b5198222fe14f99c11).